README.md

```diff
@@ -64,14 +44,19 @@ For general use, use the `main` branch along with [version 2.15.4 of the CodeQL
         use 'codeql resolve qlpacks' and 'codeql resolve languages'.
     ```
 
-1. Install CodeQL Packages using `codeql pack install`
+1. Install CodeQL Packages 
+    
+    For WHCP_21H2 and WHCP_22H2 branches:
+ 
+    **NOTE** Visual Studio 17.8 broke compatibility with the older versions of CodeQL used in the WHCP_21H2 and WHCP_22H2 branches. CodeQL 2.15.4 has been validated for use with WHCP 21H2 and WHCP 22H2. If using Visual Studio version 17.8+, follow the steps for ALL OTHER BRANCHES, below. **Make sure to remove the CodeQL submodule if it exists in your repo.** CodeQL might try to use the queries in the submodule by default which will cause errors because of mismatched versions.
 
-   For WHCP BRANCHES: Skip this step.
+    The WHCP branches WHCP_21H2 and WHCP_22H2 use the CodeQL repository as a submodule instead of using CodeQL packs. When using these branches, make sure to update submodules. 
 
-   For MAIN AND DEVELOPMENT BRANCHES use:
+    For ALL OTHER BRANCHES run the following commands:
 
-   ```
-    D:\codeql-home\codeql>codeql pack install D:\codeql-home\Windows-Driver-Developer-Supplemental-Tools\src
+    ```
+    codeql pack download microsoft/windows-drivers
+    codeql pack install <package install location>\<package name>\<package version>\
```

Comment:
Why is the codeql pack install command left with to-be-filled arguments? What is the expected action the user should take here? (i.e. are they installing the dependencies for our pack?)